doc/nrf/releases/release-notes-2.2.0.rst

```diff
@@ -0,0 +1,948 @@
+    The figure now correctly shows the interaction with the Bluetooth modules.
+  * An issue with Simple Management Protocol (SMP) not advertising in the CIS mode.
+  * An issue with the ``mcumgr`` command being unable to receive in the BIS mode.
+  * The :ref:`nrf53_audio_app_porting_guide` section in the documentation did not mention long-pressing **BTN 4** while resetting the development kit to start DFU.
+    This has now been added to the documentation.
+
+nRF Machine Learning (Edge Impulse)
+-----------------------------------
+
+* Removed the support for Thingy:52 (``thingy52_nrf52832``).
+
+nRF Desktop
+-----------
+
+* Added:
+
+  * :ref:`nrf_desktop_fast_pair_app`.
+    The module is used in configurations that integrate Google `Fast Pair`_.
+  * :ref:`CONFIG_DESKTOP_LED_STATE_DEF_PATH <config_desktop_app_options>`.
+    The option can be used to specify the file defining the used LED effects.
+  * The application configurations that enable `Fast Pair`_.
+    See nRF Desktop's :ref:`nrf_desktop_bluetooth_guide_fast_pair` documentation section for details.
+
+  * :ref:`CONFIG_DESKTOP_USB_REMOTE_WAKEUP <config_desktop_app_options>` Kconfig option for :ref:`nrf_desktop_usb_state`.
+    The option enables the USB wakeup functionality in the application.
+    The option selects :kconfig:option:`CONFIG_USB_DEVICE_REMOTE_WAKEUP`.
+
+  * Application-specific Kconfig options to simplify the configuration.
+    Part of an application Kconfig configuration that is common for the selected HID device type is introduced as overlays for default Kconfig values.
+    See :ref:`nrf_desktop_porting_guide` for details.
+
+* Updated:
+
+  * The ``CONFIG_DESKTOP_BLE_SCANNING_ENABLE`` Kconfig option has been renamed to :ref:`CONFIG_DESKTOP_BLE_SCAN_ENABLE <config_desktop_app_options>`.
+  * The UUID16 values of GATT Human Interface Device Service (HIDS) and GATT Battery Service (BAS) have been moved from advertising data to scan response data.
+  * The :kconfig:option:`CONFIG_BT_GATT_SERVICE_CHANGED` Kconfig option is disabled on nRF Desktop dongles to reduce memory footprint.
+  * The :kconfig:option:`CONFIG_BT_ID_UNPAIR_MATCHING_BONDS` is now enabled by default.
+    This is done to pass the Fast Pair Validator's end-to-end integration tests and to improve the user experience during the erase advertising procedure.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <sample>`, including protocol-related samples.
+For lists of protocol-specific changes, see `Protocols`_.
+
+Bluetooth samples
+-----------------
+
+* Added the :ref:`peripheral_cgms` sample.
+
+* :ref:`ble_throughput` sample:
+
+  * Added terminal commands for selecting the role.
+  * Updated the ASCII art used for showing progress to feature the current Nordic Semiconductor logo.
+
+* :ref:`peripheral_fast_pair` sample:
+
+  * Added:
+
+    * Bond removal functionality.
+    * Battery information to demonstrate the Fast Pair Battery Notification extension.
+
```

Comment:
```suggestion
* TX power correction value to align the advertised TX power with the Fast Pair expectations.

```

Reply:
We've fixed a bug in this PR: https://github.com/nrfconnect/sdk-nrf/pull/9573

Reply:
Added